### PR TITLE
Fixes #371 - LocalAppEngineOutputLineListener should be renamed, moved to the util bundle and #414 - Move com.google.cloud.tools.eclipse.util from localserver to util package

### LINKED ISSUE #371: LocalAppEngineOutputLineListener should be renamed, moved to the util bundle
This class looks like a good base implementation to use in other bundles as well, so it should be moved to the util bundle.

### PR DESCRIPTION
**Pull request by akerekes**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -7,6 +7,10 @@ Bundle-Vendor: Google, Inc.
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.7
 Bundle-ActivationPolicy: lazy
-Export-Package: com.google.cloud.tools.eclipse.util.io,
+Export-Package: com.google.cloud.tools.eclipse.util,
+ com.google.cloud.tools.eclipse.util.io,
  com.google.cloud.tools.eclipse.util.status
 Require-Bundle: org.eclipse.core.runtime
+Import-Package: com.google.cloud.tools.appengine.cloudsdk.process,
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF]: why do we need to import this? I would expect the util package not to have any dependencies on our other non-util packages.

**Comment by elharo** [comment]
If you're having trouble with this, try moving one class at a time, starting with the simplest.

File: plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF (hunk 1 above)
Comment (akerekes) [inline plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF]: relocated to sdk.ui bundle

File: plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/DefaultOutputLineListener.java [hunk 2]
```diff
@@ -10,16 +10,16 @@
...
  *******************************************************************************/
-package com.google.cloud.tools.eclipse.appengine.localserver.server;
+package com.google.cloud.tools.eclipse.sdk.ui;
 
 import org.eclipse.ui.console.MessageConsoleStream;
 
 import com.google.cloud.tools.appengine.cloudsdk.process.ProcessOutputLineListener;
 
-public class LocalAppEngineOutputLineListener implements ProcessOutputLineListener {
+public class DefaultOutputLineListener implements ProcessOutputLineListener {
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/DefaultOutputLineListener.java]: This name doesn't convey what it does.  `OutputStreamLineWriter`?  `MessageConsoleLineWriter`?

**Comment by briandealwis** [comment]
LGTM modulo the `DefaultOutputLineListener` class name

File: plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.util/META-INF/MANIFEST.MF]: do you still need to import this?

**Pull request merged**